### DISCUSSION TITLE
Vert.x Gateway and PostgreSQL

### DISCUSSION THREAD

**Discussion by yetanothern** [apiman/apiman#1841] [Q&A]
Is it possible to setup vert.x gateway with postgresql (not es) as on wildfly?

**Comment by msavy** [comment]
It should be possible, yes. If not, then perhaps there are some config or technical glitches that need ironing out. Have you tried it?

**Reply by yetanothern** [reply]
Didn't try yet. The question is appeared bc there is no config exapmples for it.
Will try it.
